rules.ini

```
@@ -159,7 +159,8 @@ Torque = \.cs\.dso$
 TyranoBuilder = (?:^|/)tyrano\.js$
 UbiArtFramework = pc(?:32)?\.ipk$
 UbisoftAnvil = (?:^|/)(?:datapc(?:64)?|gamedatapc_00)\.forge$
-Unigine = (?:^|/)(?:|lib)Unigine_x(?:86|64)\.(?:dylib|dll|so)$
+Unigine[] = (?:^|/)(?:|lib)Unigine_x(?:86|64)\.(?:dylib|dll|so)$
```

```suggestion
Unigine[] = (?:^|/)Unigine_x(?:86|64)\.dll$
```
let's drop these while we're at it, they don't bring any match that isn't already matched by the dll